core/domain/subscription_services.py

```diff
@@ -187,12 +215,14 @@ def get_collection_ids_subscribed_to(user_id):
 
 
 def get_last_seen_notifications_msec(user_id):
-    """Returns the last time, in milliseconds since the Epoch, when the user
-    checked their notifications in the dashboard page or the notifications
-    dropdown.
-
-    If the user has never checked the dashboard page or the notifications
-    dropdown, returns None.
+    """
+    Args:
+        user_id: str. User ID.
+    Returns:
+        str or None. The last time (in msecs since the Epoch), when the user
```

Comment:
How did you determine the type of the return value?

Reply:
I have looked at get_time_in_millisecs() method inside utils. I should return a floating point value. Earlier I misinterpreted that.